actions/action.go

```diff
@@ -137,16 +137,30 @@ func LoadActions(ctx context.Context, source Source) ([]*Action, error) {
 				return fmt.Errorf("parsing file %s: %w", addr, err)
 			}
 			actions[ii] = action
-
 			return nil
 		})
 	}
 	if err := errGroup.Wait(); err != nil {
 		return nil, err
 	}
+	if err := validateActions(actions); err != nil {
+		return nil, err
+	}
 	return actions, nil
 }
 
+// validateActions verify we do not two actions with the same name
+func validateActions(actions []*Action) error {
```

Comment:
Don't forget the validation about a single hook per action.

Reply:
I wish to remove this restriction soon, just need to address the behavior we like